test/functional/wallet_importdescriptors.py

```diff
@@ -98,6 +98,15 @@ def run_test(self):
                      labels=["Descriptor import test"])
         assert_equal(w1.getwalletinfo()['keypoolsize'], 0)
 
+        # Check persistence of data and that loading works correctly
+        w1.unloadwallet()
+        self.nodes[1].loadwallet('w1')
+        test_address(w1,
+                     key.p2pkh_addr,
+                     solvable=True,
+                     ismine=True,
+                     labels=["Descriptor import test"])
```

Comment:
why is this ordering different?

Reply:
The mine purpose is import+test+unload+load and retest address again.

We don't have required code snippet due to segwit. DNM. So, I re-implemented that test (unload / load & test) for another imported descriptor.